adapters/appnexus/appnexus.go

```diff
@@ -162,7 +163,7 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.E
 	reqExt.Appnexus.IsAMP = isAMP
 	reqExt.Appnexus.HeaderBiddingSource = a.hbSource + isVIDEO
 
-	imps := request.Imp
+	imps := requestCopy.Imp
```

Comment:
`requestCopy` is already a copy. So no need to again create a new copy by assigning value to `imps`

Reply:
addressed in [eae473c](https://github.com/prebid/prebid-server/pull/2635/commits/eae473c30e193c5f1fd16e65eb37dba9ae962818)